db/seeds.rb.example

```diff
@@ -0,0 +1,20 @@
+# Replace <your gmail name> with your gmail name below.
+
+Community.create!([
+  {name: "My Community", abbrv: "MC"}
+])
+
+Household.create!([
+  {unit_num: "0", community_id: 1, name: "Admins", deactivated_at: nil}
+])
+
+User.create!([
+  {email: "<your gmail name>@gmail.com", google_email: "<your gmail name>@gmail.com", first_name: "Alice", last_name: "Admin", mobile_phone: "17345551212", household_id: 1, admin: true}
+])
+
+Resource.create!([
+  {name: "Dining Room", community_id: 1, default_calendar_view: "week", meal_abbrv: "CH", hidden: false}
+])
+
+
+
```

Comment:
The MESS code files I see don't have a final newline (which would produce one trailing blank line).  That is, in the editor, the lowest place I can place a cursor is at the end of the last line that has content.  Is that what you intend?